test/consistent/chain/chain_test.go

```diff
@@ -68,7 +68,7 @@ func TestChainWriteSEQUENTIAL(t *testing.T) {
 	if _, err := chain.GetTail().Read(context.TODO(), "k1"); err == nil {
 		t.Fatalf("tail failed is supposed not to find the key")
 	}
-	time.Sleep(5 * time.Second)
+	time.Sleep(10 * time.Second)
```

Comment:
what's the reason of this change?

Reply:
Ran several times locally but failed once since the write is async, it might take more time than expected, especially in git servers.